org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java

```diff
@@ -368,6 +369,10 @@ private synchronized void doNodeInitialisation() {
                     continue;
                 }
 
+                if (channel.getConfiguration().get(ZclReportingConfig.CONFIG_POLLING) == null) {
+                    channel.getConfiguration().put(ZclReportingConfig.CONFIG_POLLING, handler.getPollingPeriod());
```

Comment:
I believe that by setting the polling period configuration for every channel, users will think that it is also configurable for every channel; but, as far as I see it, the configuration is currently only applied for switch level and switch on off converters.

Shouldn't the polling period config value only be set for the channels where it can actually be configured?

Reply:
Sorry - i missed this comment.

All converters return a polling period, and most should work shouldn't they? Do you mean that this config handler is only (currently) applied to level and on/off channels? If that's what you mean, then this config will currently only show for these channels so I'm not sure what you mean by this comment - sorry.

Reply:
I think I see your point now; the config is set, but not used, but this will not hurt anybody... Thanks for your response on this.